server/core/SqlQueryBuilder.js

```diff
@@ -292,6 +292,12 @@ class SqlQueryBuilder {
         case 'notNull':
             return `${field} IS NOT NULL`;
 
+        case 'empty':
+            return `${field} = ' '`;
```

Comment:
It's strange to test the emptiness of a string like this (comparing it to a single space character). It looks like it works only because of MySQL's special treatment of trailing spaces: https://dev.mysql.com/doc/refman/8.0/en/charset-binary-collations.html#charset-binary-collations-trailing-space-comparisons
But depending on the collation, `'' = ' '` could be either true or false.  Moreover, it looks like it doesn't work at all with PostgreSQL. (`'' = ' '` returns false, except for type `char(n)`)

Depending on what you want, you should either:
1) use `TRIM(field) = ''` which will remove any leading/trailing space from field and then compare it to an empty string (no collation magic involved)
2) use `field = ''` (empty string) and let the collation do its magic

Reply:
I didn't realize this, I use TRIM like you suggest it